mbtools/validate_mbz_html.py

```diff
@@ -66,6 +70,35 @@ def validate_mbz(mbz_path, include_styles=True, include_questionbank=False):
     return violations
 
 
+def validate_html(html_dir, include_styles=True):
+    all_files = []
+    for path, dirs, files in os.walk(html_dir):
```

Comment:
It might be nice to scope to just `*.html` files just in case we end up having a mix of file types in directories at some point. I don't know if `os.walk` allows that. If not, maybe we can use `glob`?